package-lock.json

```diff
@@ -1,8 +1,6950 @@
 {
   "name": "sbt-accessibility-linter",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
my npm version added this - seems it's some optimisation for resolving dependencies, apparently backward compatible